sphinx-docs/faq.rst

```diff
@@ -0,0 +1,191 @@
+Frequently Asked Questions
+==========================
+
+How do I install KA Lite?
+-------------------------
+
+Information on how to install KA Lite is available via our :doc:`user guides </usermanual/userman_main>`.
+
+How much does KA Lite cost to install?
+--------------------------------------
+
+It is **FREE**.
```

Comment:
Might want to add an additional short sentence about us being free (gratis -- 0 USD) and free (open source, free to modify etc.) and that most of our content is open-licensed too.